extension-manifest-v3/src/background/healthcheck.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { captureException } from '/utils/errors.js';
+
+async function healthcheck() {
+  try {
+    await indexedDB.databases();
+    await chrome.storage.local.get(['test']);
+  } catch (error) {
+    console.error(error);
+    captureException(error);
+  }
+}
+
+setTimeout(healthcheck, 10000);
```

Comment:
I don't like calling it for 99% of users, which does not have that problem. We should add a `captureException` at least only in `engines.js`, where `loadFromStorage` or `saveToStarge` fails, meaning that `indexedDB` is broken.